dashboard/test/test_helper.rb

```diff
@@ -124,6 +124,18 @@ def assert_does_not_create(*args)
     end
   end
 
+  def assert_destroys(*args)
+    assert_difference(args.collect(&:to_s).collect {|class_name| "#{class_name}.count"}, -1) do
+      yield
+    end
+  end
+
+  def assert_does_not_destroy(*args)
+    assert_no_difference(args.collect(&:to_s).collect {|class_name| "#{class_name}.count"}) do
+      yield
+    end
+  end
```

Comment:
I prefer `assert_no_difference`, because it implies that the block does not create or destroy, whereas `assert_does_not_create` is not clear whether or not it would allow a destroy.